etcdserver/etcdserverpb/raft_internal.proto

```diff
@@ -35,7 +35,8 @@ message InternalRaftRequest {
   AuthRoleGrantRequest auth_role_grant = 17;
   AuthenticateRequest authenticate = 18;
 
-  AlarmRequest alarm = 19;
+  string userID = 19;
```

Comment:
username? also this should probably be header of all v3 requests instead of a field inside internal raft request.